lib/sles4sap/sap_deployment_automation_framework/basetest.pm

```diff
@@ -96,6 +98,7 @@ sub full_cleanup {
 }
 
 sub post_fail_hook {
+    my ($self, $run_args) = @_;
```

Comment:
Could you please explain why and how `$run_args` come in use? What data is expected to provide? From/To which other entity? Which problem is going to solve?